source/geometries/Next100Shielding.cc

```diff
@@ -36,27 +36,25 @@ namespace nexus {
     GeometryBase(),
 
     // Shielding internal dimensions
-    shield_x_ (155.  * cm),
-    shield_y_ (225.6 * cm),
-    shield_z_ (258.5 * cm), // 253.0 * cm before May 12, 2017
+    shield_x_ (158.  * cm),
+    shield_y_ (166.  * cm),
+    shield_z_ (259.4 * cm),
 
     //Steel Structure
-    beam_base_thickness_ (4. *mm),
-    lateral_z_separation_ (1006. *mm), //distance between the two lateral beams
-    roof_z_separation_ (755. *mm), //distance between 1st and 2nd roof beams
-    front_x_separation_ (154.*mm), //distance between the two front beams
+    beam_base_thickness_ (4.   * mm),
+    lateral_z_separation_(1010.* mm), //distance between the two lateral beams
```

Comment:
Are these the distances between the centres of the beams or between the sides? Please, specify. If it's the second case, the union of solids below needs to be reviewed.

Reply:
From Jordi's drawings I think it is between the beam centers. If it were not the case, the total distance (see lateral view in slide 5) would not sum up, since we had to add two times the beam thickness.